test/versioned/cassandra-driver/query.tap.js

```diff
@@ -25,6 +26,26 @@ const client = new cassandra.Client({
   localDataCenter: 'datacenter1'
 })
 
+const colValArr = ['Jim', 'Bob', 'Joe']
+const pkValArr = [111, 222, 333]
+let insQuery = 'INSERT INTO ' + KS + '.' + FAM + ' (' + PK + ',' + COL
+insQuery += ') VALUES(?, ?);'
+
+const insArr = [
+  { query: insQuery, params: [pkValArr[0], colValArr[0]] },
+  { query: insQuery, params: [pkValArr[1], colValArr[1]] },
+  { query: insQuery, params: [pkValArr[2], colValArr[2]] }
+]
+
+const hints = [
+  ['int', 'varchar'],
+  ['int', 'varchar'],
+  ['int', 'varchar']
+]
+
+let selQuery = 'SELECT * FROM ' + KS + '.' + FAM + ' WHERE '
+selQuery += PK + ' = 111;'
```

Comment:
Do any of these get mutated by tests? Or do the tests merely read the values/components?

Reply:
The tests just insert and then read the values